com.unity.shadergraph/Documentation~/Parallax-Occlusion-Mapping-Node.md

```diff
@@ -2,30 +2,29 @@
+You can use the Parallax Occlusion Mapping (POM) node to create a parallax effect that displaces a material's UVs and depth to create the illusion of depth inside that material.
 
-If you experience texture sampling errors while using this node in a graph which includes Custom Function Nodes or Sub Graphs, it might be possible to resolve these errors by upgrading to version 10.3 or later.
-
-When you assign the same Texture2D to a Parallax Occlusion Mapping Node and a Sample Texture 2D Node, you need to avoid transforming the UV coordinates twice. To prevent this, connect the Split Texture Transform Node’s **Texture Only** port to the Sample Texture 2D Node’s UV port.
-
-![](images/node-parallaxocclusionmapping.PNG)
+If you experience texture sampling errors while using this node in a graph that includes Custom Function Nodes or Sub Graphs, it might be possible to resolve these errors by upgrading to Shader Graph version 10.3 or later.
 
+When you assign the same Texture2D to a POM node and a Sample Texture 2D node, you need to avoid transforming the UV coordinates twice. To prevent this, connect the Split Texture Transform node’s **Texture Only** port to the Sample Texture 2D Node’s UV port.
 
 ## Ports
 
 | Name | **Direction** | Type | Description |
 | --- | --- | --- | --- |
 | **Heightmap** | Input | Texture2D | The Texture that specifies the depth of the displacement. |
-| **Heightmap Sampler** | Input | Sampler State | The Sampler to sample **Heightmap** with. |
+| **Heightmap Sampler** | Input | Sampler State | The Sampler to sample Heightmap with. |
 | **Amplitude** | Input | Float | A multiplier to apply to the height of the Heightmap (in centimeters). |
 | **Steps** | Input | Float | The number of steps that the linear search of the algorithm performs. |
 | **UVs** | Input | Vector2 | The UVs that the sampler uses to sample the Texture. |
-| **Lod** | Input | Float | The level of detail to use to sample the **Heightmap**.
-| **Lod Threshold** | Input | Float | The **Heightmap** mip level where the Parallax Occlusion Mapping effect begins to fade out. This is equivalent to the **Fading Mip Level Start** property in the High Definition Render Pipeline's (HDRP) [Lit Material](Lit-Shader.md). |
-| **Depth Offset** | Output |Float | The offset to apply to the depth buffer to produce the illusion of depth. Connect this output to the **Depth Offset** on the Master Node to enable effects that rely on the depth buffer, such as shadows and screen space ambient occlusion. |
+| **Tiling** | Input | Vector2 | The tiling to apply to the input UVs. |
+| **Offset**| Input | Vector2 | The offset to apply to the input UVs. |
+| **Primitive Size** | Vector2 | Float | Size of the UV space in object space. For example, a Unity built-in Plane mesh has a primitive size of (10,10). |
+| **Lod** | Input | Float | The level of detail to use to sample the Heightmap. |
```

Comment:
Is there a limit on the floats that can be inputted here? What guidance can we provide on what values a user should use?

Reply:
The descriptions here are taken of other existing node for coherency, and the values are not bound, except for the LOD which should always be positive, but it's [not mentioned either here](https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.shadergraph/Documentation~/Sample-Texture-2D-LOD-Node.md).